### PR TITLE
feat: homepage 1st commit

### PR DESCRIPTION
**Pull request by ankit9015**
This is the homepage of the component library. Thanks for reviewing it.

https://user-images.githubusercontent.com/67504902/152626682-a362fa2e-6d66-4dd9-86f5-0de92fd3a32e.mp4

### CODE REVIEW

File: reset.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
```
Comment (Raghav888) [inline reset.css:9]: please use good names for classes.

**Review by Raghav888 — APPROVED** [review]
In next PR please change all class name (use meaning names)

File: reset.css (hunk 1 above)
Comment (ankit9015) [reply reset.css:9]: Its a reset.css I used from: https://meyerweb.com/eric/tools/css/reset/  I will edit it later

File: main.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+
+main {
+    display: flex;
+    flex-direction: row;
+    font-size: 1.2rem;
+}
+
+aside {
```
Comment (kotalwar-krishna) [inline main.css:54]: use classes insted of using tags  its best practice.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ display: flex;
+ flex-direction: column;
+ justify-content: center;
+ gap: 1rem;
+ height: 100vh;
+ width: 25vw;
+ align-items: center;
+}
+
+section {
```
Comment (kotalwar-krishna) [inline main.css:66]: same thing if you use section any where afterwords and you want diffrent style for that sestion then its difficult so use class so its best practice

**Review by kotalwar-krishna — COMMENTED** [review]
please do use class instead of tags directly its best practice as per my understanding.

File: main.css (hunk 2 above)
Comment (ankit9015) [reply main.css:54]: This is in reset.css file, we use reset.css to modify/remove any default CSS added by the browser.  I want this to be universal for all the mentioned selectors which is why I am not using any class selector.

Comment (ankit9015) [reply main.css:54]: I have used class selectors in main.css which are specific for the block

Comment (kotalwar-krishna) [reply main.css:54]: ok

**Review by kotalwar-krishna — APPROVED** [review]

**Pull request merged**